src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -78,244 +35,409 @@ function createModalStackNavigator(screens) {
 }
 
 const IOUBillStackNavigator = createModalStackNavigator([{
-    Component: IOUBillPage,
+    getComponent: () => {
+        const IOUBillPage = require('../../../pages/iou/IOUBillPage').default;
+        return IOUBillPage;
+    },
     name: 'IOU_Bill_Root',
```

Comment:
NAB: Should we leave some comment at the top of this file explaining why we prefer the require construction? I think it would be handy for any engineer going over to be able to see and get some context on why, then potentially they can find this PR if they want even deeper look. 

But it would be nice to have some comment in this file as well @Szymon20000

Reply:
Sth Like:
"We use getComponent/require syntax so that file used by screens are not loaded until we need them. You can read more in pr description https://github.com/Expensify/App/pull/10986". 
Who should merge this pr?
me or reviewer?

Reply:
@Szymon20000 Yes I think that would be a sufficient description but I would not include the PR link. One can always find that from the git blame.

The reviewers will merge, lets see what @marcaaron and @luacmartins have to say here